control-plane/roles/logging/tasks/main.yaml

```
@@ -50,7 +50,13 @@
     content: "{{ logging_gardener_virtual_garden_kubeconfig | b64decode }}"
   when: logging_gardener_enabled
 
-- name: Loop over Gardener seeds
+- name: Extract seed names into a new list so name can be the item in the next loop
+  set_fact:
+    gardener_seed_names: "{{ gardener_shooted_seeds | map(attribute='name') | list }}"
```

Make sure that this role works without having Gardener in place. Gardener is only an option when you deploy metal-stack.

```suggestion
    gardener_seed_names: "{{ gardener_shooted_seeds | map(attribute='name') | list }}"
  when: logging_gardener_enabled
```